nautobot/project-static/js/nav_menu.js

```diff
@@ -237,8 +237,9 @@ document.addEventListener('DOMContentLoaded', function() {
     if (activeLink) {
         let previouslyClickedLink = document.querySelector('.navbar-fixed-left .navbar-nav > .dropdown > .nav-dropdown-menu > li > a[href="' + activeLink + '"]');
         let currentLocation = window.location.pathname + window.location.search;
+        let previouslyClickedLinkNoSearch = previouslyClickedLink.getAttribute('href').split('?')[0];
 
-        if (previouslyClickedLink && currentLocation.includes(previouslyClickedLink.getAttribute('href'))) {
+        if (previouslyClickedLink && (currentLocation.includes(previouslyClickedLink.getAttribute('href')) || currentLocation.includes(previouslyClickedLinkNoSearch))) {
```

Comment:
Is this a bad merge conflict resolution or is this required for your PR?

Reply:
Required.  This keeps the NavMenuItem focused (in orange) in the navbar when navigating to a detail view from the list view. This is because the Detail view will drop the `search` string.